pandas/io/excel/_base.py

```diff
@@ -82,7 +82,7 @@
     * If None, then parse all columns.
     * If int, then indicates last column to be parsed.
 
-      .. deprecated:: 0.24.0
+    .. deprecated:: 0.24.0
```

Comment:
I don't think this change is fully correct, as the "deprecated" notion is specific to this bullet item, not to the general keyword 

(the same might be true for the other occurrences here)

Reply:
Hi, thanks for reviewing. While i agree that it would be better to have the warning indented, that breaks the validation, raising error `PR09 description should end in .` I have experimented with different amounts of indentation but this is the only one that doesn't raise a warning.

The way the docs render without the two spaces, seen below, I think are sufficiently clear about what the warning is referring to but I'm open to other solutions. I just haven't found any.

![Screenshot_2019-11-12_10-02-53](https://user-images.githubusercontent.com/38143549/68682803-b36da600-0533-11ea-8eb9-911603b1ad30.png)

Reply:
> While i agree that it would be better to have the warning indented, that breaks the validation, raising error PR09 description should end in .

Then I would consider that a bug in the validation script that could be fixed instead?

Reply:
maybe @datapythonista can speak to exactly what is being worked on in the checks, I know there are some changes in the works to handle bulleted lists without leading text.